{
    "type": "bundle",
    "id": "bundle--ab63799f-39e7-4836-8a71-27c90e3c0344",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--ec56a3e0-0672-4c88-8844-0d3cd06f4393",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2024-05-04T00:28:04.383828Z",
            "modified": "2024-05-04T00:28:04.383828Z",
            "name": "CVE-2022-48670",
            "description": "In the Linux kernel, the following vulnerability has been resolved:\n\npeci: cpu: Fix use-after-free in adev_release()\n\nWhen auxiliary_device_add() returns an error, auxiliary_device_uninit()\nis called, which causes refcount for device to be decremented and\n.release callback will be triggered.\n\nBecause adev_release() re-calls auxiliary_device_uninit(), it will cause\nuse-after-free:\n[ 1269.455172] WARNING: CPU: 0 PID: 14267 at lib/refcount.c:28 refcount_warn_saturate+0x110/0x15\n[ 1269.464007] refcount_t: underflow; use-after-free.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2022-48670"
                }
            ]
        }
    ]
}